### DISCUSSION TITLE
Stirling PDF - OCR "disabled"

### DISCUSSION THREAD

**Discussion by DeepCoreSystem** [tteck/Proxmox#2538] [Help Needed] [answered]
I installed last weekend the Stirling PDF LXC container on Proxmox. Now I wanted to do some OCR and received the following screen:

![grafik](https://github.com/tteck/Proxmox/assets/3703938/19809266-4004-46f7-ae56-85464a2adb31)

Before, I used to use an Alpine Linux Docker container with the docker setup directly from Stirling PDF, and there the OCR was available.

Following the [link displayed](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/HowToUseOCR.md#non-docker) you find instructions to add the support for OCR.

An idea would be to allow the user of the script to choose the OCR components he wants

**Comment by DeepCoreSystem** [comment]
Applying the update script didn't help...

```
   _____ __  _      ___                 ____  ____  ______
  / ___// /_(_)____/ (_)___  ____ _    / __ \/ __ \/ ____/
  \__ \/ __/ / ___/ / / __ \/ __ `/___/ /_/ / / / / /_
 ___/ / /_/ / /  / / / / / / /_/ /___/ ____/ /_/ / __/
/____/\__/_/_/  /_/_/_/ /_/\__, /   /_/   /_____/_/
                          /____/
 ✓ Updated Stirling-PDF to v0.21.0
root@stirling-pdf:~#
```

And applying the steps from the Stirling PDF documentation didn't help either:

```
root@stirling-pdf:~# apt install tesseract-ocr-deu tesseract-ocr-eng tesseract-ocr-fra
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
tesseract-ocr-deu is already the newest version (1:4.1.0-2).
tesseract-ocr-eng is already the newest version (1:4.1.0-2).
tesseract-ocr-fra is already the newest version (1:4.1.0-2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
root@stirling-pdf:~#
```

**Comment by Narkowii** [comment]
same issue here

**Comment by DeepCoreSystem** [comment, marked as answer]
So, it seems that the needed OCR data is in the wrong directory. On my LXC it is in /usr/share/tesseract-ocr/5 ...

A quick and dirty `cp -r /usr/share/tesseract-ocr/5/* /usr/share/` did resolve it for me and the moment

**Comment by DeepCoreSystem** [comment]
See my last comment with a quick and dirty "hack" ...

Am 22.02.2024 um 14:04 schrieb Narkowii:

**Reply by Narkowii** [reply]
nice, thanks !

**Comment by tteck** [comment]
```
ln -s /usr/share/tesseract-ocr/5/tessdata/ /usr/share/tessdata
```
has been added to the script
